WebsiteGreetz/src/test/java/WebsiteTest.java

```diff
@@ -24,56 +22,53 @@ private void logIn(WebDriver driver) throws InterruptedException {
         Thread.sleep(5000);
     }
 
-    private boolean findFavorite(WebDriver driver, int countOfFavorites) {
-        if (driver.findElements(By.className("center full-size")).size() > countOfFavorites)
-            return true;
+    private boolean findFavorite(int countOfFavorites) throws InterruptedException {
+        Thread.sleep(5000);
+        return driver.findElements(By.className("full-size")).size() > countOfFavorites;
+    }
 
-        return false;
+    private boolean find5and90euro() throws InterruptedException{
+        return driver.findElement(By.xpath("//div[@class='price-total' and contains(text(), '5,90')]")).isDisplayed();
     }
 
     @Test
     public void addAndCheckFavorite() throws InterruptedException {
         System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
         driver = new ChromeDriver();
         logIn(driver);
-
-        //ավելացնենք ֆավորիտ
-
         driver.get("https://www.greetz.nl/ballonnen/denken-aan");
         Thread.sleep(3000);
-
-        WebElement item = driver.findElement(By.xpath("//a[@data-id='1142804226']/.."));
-        WebElement favoriteItem = item.findElement(By.className("b-products-grid__item-action"));
+        WebElement item = driver.findElement(By.xpath("//a[@data-id='1142804226']"));
+        WebElement favoriteItem = item.findElement(By.xpath("//parent::div//child::a[@class='b-products-grid__item-action']"));
         favoriteItem.click();
-
-        //ստուգենք ավելացավ, թե չէ
-
         WebElement sideMenu = driver.findElement(By.xpath("//i[@class='page-header__navigation-item-icon b-icon b-icon-hamburger']"));
         sideMenu.click();
-
         WebElement favorite = driver.findElement(By.xpath("//span[@class='b-list--item-subject' and contains(text(), 'Favorieten')]"));
         favorite.click();
-
-        //int countOfFavorites = 0;
-        //Assert.assertTrue(findFavorite(driver, countOfFavorites));
-
-
-
+        int countOfFavorites = 0;
+        Assert.assertTrue(findFavorite(countOfFavorites));
+        //մեկ ու մեջ ա աշխատելու, մի անգամ նշումա, հաջորդ անգամը ՝ հանում
```

Comment:
of course, you can and should handle that case, if the item is already in your favorites, another workflow can be, or simply you can remove it from the favorites, then add it once more. let's say you should have a fully working test method, in real life, it's not acceptable to expect TC fail in any 2nd try